ports.go

```diff
@@ -54,6 +54,9 @@ func ListPorts(client *gophercloud.ServiceClient) <-chan Resource {
 		if err := ports.List(client, nil).EachPage(func(page pagination.Page) (bool, error) {
 			resources, err := ports.ExtractPorts(page)
 			for i := range resources {
+				if strings.Contains(resources[i].DeviceID, "ovnmeta") {
```

Comment:
@pierreprinetti I saw that all the Filters defined here are generic and [could be used by any resource.](https://github.com/shiftstack/prune/blob/main/filter.go) that is why I proposed adding the skip of Ports here given DeviceID is specific to Ports.
WDYT?